content/posts/clean-your-backlog.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Clean your backlog"
+date: 2021-11-10
+language: en
+author: Dimiter Petrov
+---
+
+Have you recently checked the size of your backlog?
+
+There is a limit to the amount of work a team can do at a given time, so new tasks get queued in a backlog. As a project grows, so does its backlog. There are new features to develop, more bugs to fix, but these tasks take longer to finish than to write down.
+
+## The problem with large backlogs
+
+Large backlogs are demoralizing. It is intimidating to realize how much is left to do.
+
+Large backlogs are overwhelming. People lose the overview. They create a second, smaller, "prioritized" backlog. Or they start tracking work through other channels: in-person meetings, email, text chat. That exacerbates the problem - now you have multiple backlogs and they all keep growing.
+
+Large backlogs stagnate. The bigger the backlog, the more information is outdated and the more maintenance is required.
+
+Large backlogs give a false sense of how much is really planned, because most items are too vague or not estimated yet.
+
+## Keep your backlog small(er)
+
+Reducing the backlog size can reduce the team's stress level. It also allows the team to focus on what's the most important. Here are some ideas to help you declutter:
+
+- **Acknowledge your backlog is too big.** They all are. Discuss that with your team and agree on what you want to do about it.
+- **Review the backlog as your priorities change.** What you though was important 4 months ago may not be as important now. Outright remove what has become irrelevant and reorder the rest to align with the new requirements.
```

Comment:
There is a small typo: `though` instead of `thought`

Reply:
Thank you! I fixed it in b7401e4